content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -1284,7 +1284,16 @@ Check out the [**Bevy 0.14 Milestone**](https://github.com/bevyengine/bevy/miles
 * **More editor experimentation:** TODO
 * **bevy_dev_tools:** TODO
 * **A revised scene format:** TODO
-* **bevy_ui improvements:** TODO
+* **bevy_ui improvements:** `bevy_ui` has its fair share of problems and limitations, [both mundane and architectural].
+But there are tangible things we can and are doing to improve this!
+An improved scene format offers an end to the boilerplate when defining layouts, [rounded] [corners] just need a little love from reviewers, and the powerful and beloved object picking from [`bevy_mod_picking`] is slated to be upstreamed for both UI and gameplay alike.
```

Comment:
```suggestion
* **bevy_ui improvements:** `bevy_ui` has its fair share of problems and limitations, [both mundane and architectural];
however, there are tangible things we can and are doing to improve this:
an improved scene format offers an end to the boilerplate when defining layouts, [rounded] [corners] just need a little love from reviewers, and the powerful and beloved object picking from [`bevy_mod_picking`] is slated to be upstreamed for both UI and gameplay alike.
```
Changes to make these lines flow together rather than feel disjointed.